hikari/events/guild_events.py

```diff
@@ -265,6 +266,12 @@ class GuildUpdateEvent(GuildEvent):
     shard: gateway_shard.GatewayShard = attr.ib(metadata={attr_extensions.SKIP_DEEP_COPY: True})
     # <<inherited docstring from ShardEvent>>.
 
+    old_guild: typing.Optional[guilds.Guild] = attr.ib()
+    """The old guild object.
+
+    This will be `builtins.None` if guild missing from the cache.
+    """
```

Comment:
Just a nickpick to make everything look better and cause less confusion. This can also be applied to all the other `old_x` fields

```suggestion
    """The old guild object.

    This will be `builtins.None` if the guild missing from the cache.
    """
```